Makefile

```diff
@@ -30,10 +30,10 @@ CONCUERROR_TESTS = ranch_concuerror:start_stop ranch_concuerror:info
 dep_ci.erlang.mk = git https://github.com/ninenines/ci.erlang.mk master
 DEP_EARLY_PLUGINS = ci.erlang.mk
 
-AUTO_CI_OTP ?= OTP-21+
+AUTO_CI_OTP ?= OTP-22+
```

Comment:
Please also update doc/src/guide/introduction.asciidoc

Reply:
Ok, I changed it.